extras/ams_wrapper/src/api/models/vehicle_detection_adas_model.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import sys
+import json
+from logger import get_logger
+from api.models.model import Model
+
+logger = get_logger(__name__)
+
+
+class VehicleDetectionAdas(Model):
+
+    def load_default_labels(self):
```

Comment:
This shall be changed when https://github.com/openvinotoolkit/model_server/pull/208/files is merged. We could define labels file path here:
https://github.com/openvinotoolkit/model_server/blob/ams/extras/ams_wrapper/src/config.py#L23

Reply:
Yes we will merge it in next iteration.